torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -337,23 +339,43 @@ std::string ToXlaString(const c10::Device& device) {
   return absl::StrCat("xla:", device.index());
 }
 
+const torch::lazy::BackendDevice* GetDefaultDevice() {
+  std::string default_device_spec =
```

Comment:
nit add `static`. `UseVirtualDevice` should not changed after default device is inited.

Reply:
Done